src/hooks/use-steps.tsx

```diff
@@ -21,21 +19,15 @@ const useSteps = ({ steps }: UseSteps) => {
   const { stepData } = useStepContext()
   const dispatch = useAppDispatch()
   const { userId } = useAppSelector((state) => state.appMain)
+  const { handleErrorAlert } = useSnackbarAlert()
 
-  const updateUser = useCallback(
-    (params?: UpdateUserParams) => userService.updateUser(userId, params!),
+  const handleUpdateUser = useCallback(
+    (params?: UpdateUserParams) => {
+      return userService.updateUser(userId, params!)
+    },
```

Comment:
```suggestion
  const handleUpdateUser = useCallback(
    (params: UpdateUserParams) => {
      return userService.updateUser(userId, params)
    },
```

We were using both params?: UpdateUserParams and params! in the same function, which is contradictory. The ? indicates that the argument is optional, while the ! forcefully asserts that it's defined.
Since we always expect params to be passed, it's cleaner and safer to make it a required parameter and remove the unnecessary non-null assertion.

Reply:
Thanks for advice) I will fix it in next issue.